multi-joins/flinksql/src/test/resources/populate-shipments.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO shipments
+VALUES ('shipment_1',   1, 'Bar Harbor',     1692815775),
```

Comment:
nit: Always a "fun" issue with timestamped events in testing. I see these timestamps are hardcoded now. What are your thoughts on some sort of "dynamic timestamp" instead? I know this changes how the assertions are evaluated, but just food for thought.

Reply:
good catch - deleted this file as it wasn't used